provider/cmd/app/app.go

```diff
@@ -97,6 +99,7 @@ func (o *Options) AddFlags(fs *pflag.FlagSet) {
 	fs.StringVar(&o.RootDir, "libvirt-provider-dir", filepath.Join(homeDir, ".libvirt-provider"), "Path to the directory libvirt-provider manages its content at.")
 
 	fs.StringVar(&o.PathSupportedMachineClasses, "supported-machine-classes", o.PathSupportedMachineClasses, "File containing supported machine classes.")
+	fs.DurationVar(&o.ResyncDurationVolumeSize, "volume-size-resync-duration", o.ResyncDurationVolumeSize, "Interval to determine volume size changes.")
```

Comment:
Shouldn't that be rather called interval instead of duration?

Reply:
maybe we can add additional information into help. It is interval between calling of function. So if check takes 5min (network issues), it can be confusing.

Reply:
> Shouldn't that be rather called interval instead of duration?

fixed the description to have it consistent

Reply:
Sorry but i prefer as @afritzler mention interval. Duration meaning resync function has to be done in specified time. It isn't semantic correctly, because this variable modified interval between 2 calls of resync function

Reply:
i am okay with current current fix. If you want to you can close issue, i am not author so i don't want to do that.